vignettes/submodel_documentation_overview.Rmd

```diff
@@ -0,0 +1,394 @@
+
+*Source*
+
+Natural adult removal rate was derived using coded wire tag analysis. Details and methodology are available [here](https://reorienting-to-recovery.github.io/fallRunDSM/articles/hatchery-analysis.html).
+
+Compiled by: James T. Peterson jt.peterson@oregonstate.edu
+
+### Adult Harvest 
+
+Adult harvest is applied in the model as a percentage of ocean harvest and a percentage of in river tributary harvest. Harvest is applied to adult escapement numbers, before adult prespawn and migratory survival is applied. In the model, this information is contained in the `ocean_harvest_percentage` & `tributary_harvest_percentage` data objects and it is input to the `fall_run_model` as part of the params list. 
+
+Base model logic sets: 
+
+- Ocean harvest 50% 
+- In river harvest 8% for Sacramento River and its tributaries
+- And gives all fish have equal likelihood of being harvested 
+
+*Source*
+
+Harvest rates were pulled from table II-1 of the [Preseason Report I Stock Abundance Analysis and Environmental Assessment Part 1 for 2022 Ocean Salmon Fishery Regulations](https://www.pcouncil.org/documents/2022/03/2022-preseason-report-i.pdf/).
+
+However additional model parameters can change harvest mode to one of the following: 
+
+- Intelligent habitat harvest
+- Intelligent cohort replacement rate harvest (CRR)
+- Age selective harvest 
+- Dry year harvest regulations 
+- Hatchery harvest only (ocean and/or tributary)
+
+For more information on these harvest strategies please see [DSM scenario harvest actions](https://reorienting-to-recovery.github.io/R2Rscenarios/articles/harvest-actions.html). 
+
+### Adult Enroute Migratory Survival 
+
+Adult enroute migratory survival is a function of migratory temperatures and whether or not the bypass is overtopped. This function also contains a calibrated intercept.  In the model, this information is contained in the following function input objects: 
+
+- data: `migratory_temp`
+- data: `bypass_overtopped`
+- coefficent: `.adult_en_route_bypass_overtopped`
+- coefficent: `.adult_en_route_migratory_temp`
+- calibrated intercept: `..surv_adult_enroute_int` 
+
+and it is input to the `fall_run_model` as part of the params list. 
+
+*Source*
+
+- `.enroute migratory_temp` coefficent from Schreck et al. (1994)
+- `.adult_en_route_bypass_overtopped`coefficient for bypass_overtopped variable, source: Expert opinion Ted Sommer, California Department of Water Resources (tributaries above bypasses only)
+- *Calibrated intercept.* 
+
+### Adult Stray 
+
+The adult straying function is a beta regression model that including the following factors that straying rates:  transport distance, return year flows, fish age, release timing, and environmental conditions (PDO). This logic was developed by Sturrock et al. (2019). Within the DSM, the `compute_adult_stray_rates` function, takes in: 
```

Comment:
"following factors that *affect* straying rates"

Reply:
updated